examples/bbcsounds_example.py

```diff
@@ -0,0 +1,47 @@
+"""
+Example on how to use the BBC iPlayer Controller
+
+"""
+# pylint: disable=invalid-name
+
+import logging
+from time import sleep
+import sys
+
+import pychromecast
+from pychromecast.controllers.bbcsounds import BbcSoundsController
+
+# Change to the name of your Chromecast
+CAST_NAME = "Lounge Video"
```

Comment:
Copy the argument handling used in the other examples to make the cast name, debug level etc. configurable from command line 👍